packages/flutter/test/material/app_bar_test.dart

```diff
@@ -1514,4 +1514,37 @@ void main() {
 
     expect(getMaterialWidget().shape, roundedRectangleBorder);
   });
+
+  testWidgets('SliverAppBar with shape', (WidgetTester tester) async {
+    const RoundedRectangleBorder roundedRectangleBorder = RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(15.0)));
```

Comment:
Break this out into multiple lines, since Flutter's style guide recommends max 80 chars per line if possible:
```
const RoundedRectangleBorder roundedRectangleBorder = RoundedRectangleBorder(
  borderRadius: BorderRadius.all(Radius.circular(15.0)),
);
```

Reply:
Could you update the test for `AppBar with shape` to be broken out in a similar fashion as well?

Reply:
sure